docs/developer/testing_framework.md

```diff
@@ -119,6 +121,14 @@ If such a function exists, the runner will check for existence of the correspond
     }
     ```
 
+=== "Integration test function"
```

Comment:
In the documentation, this box is having now many windows with long titles and they do not fit anymore in the available horizontal space leading to a horizontal scrolling. I suggest to remove the word `function` from all of them, changing `Setup|Teardown function` to `Test setup|teardown` or simply `Setup` and `Teardown` as you prefer.